projects/boringssl/Dockerfile

```diff
@@ -15,7 +15,10 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y cmake ninja-build golang
-
+RUN apt-get update && apt-get install -y cmake ninja-build golang curl wget \
```

Comment:
Are both `curl` and `wget` needed here? They're both client libraries for fetching resources, and it's not clear to me which dependencies there are.

Reply:
I deleted curl